lua/neotest-go/color.lua

```diff
@@ -1,16 +1,136 @@
 local M = {}
 
--- Highlight some parts of the test output
-function M.highlight_output(output)
+local fmt = string.format
+local colors_patterns = require("neotest-go.patterns").colors
+
+local term = os.getenv("COLORTERM")
+local fullcolor = (term == "truecolor") or (term == "24bit")
+
+local function slice(tbl, first, last, step)
+  local sliced = {}
+  for i = first or 1, last or #tbl, step or 1 do
+    sliced[#sliced + 1] = tbl[i]
+  end
+  return sliced
+end
+
+---Escape characters that have a special meaning in patterns
+---@param text string
+---@return string
+local function escape(text)
```

Comment:
will `vim.pesc` be a suitable replacement for this custom function?

Reply:
Yes